docs/support/faq/how-to-use-pouta-for-remote-desktop.md

```diff
@@ -0,0 +1,147 @@
+# How to use cPouta for Remote Desktop
+
+This chapter will go through how to set up a remote desktop with noVNC and
+ssh-tunnel on a non-GPU flavor in c- and ePouta. We are using noVNC because
+it allows us to use our browser to access the desktop. If you would like to have
+instruction how to use a GPU instance for rendering you can find instructions
+here [How to use cPouta GPU for rendering](how-to-use-cpouta-gpu-for-rendering.md)
+
+[TOC]
+
+## Preparations
+We will utilize the following technologies to install the remote desktop:
+
+  - A standard-flavor for example standard.medium 
+  - CentOS-7 images (In Pouta it has already the epel-repository installed)
+  - snapd to install the noVNC
```

Comment:
snapd is some ubuntu tool?

Reply:
That is my understanding but epel is pretty magical in that way.

Reply:
but is using snap any better than yum? dont you need to install snap to then install novnc?

Reply:
Mostly because that was the I tried first. But snappy have a new version of novnc with **full screen support** (at least I did not find it in the yum version). I think full screen support is a killer feature.